docs/content/reference/linter.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Porter Lint Messages
+description: Describes the error and warning messages returned by [porter Lint command](https://getporter.org/cli/porter_lint)
+---
+
+* [exec-100](#exec-100)
+* [porter-100](https://getporter.org/reference/linter/#porter-100)
```

Comment:
Let's leave this off and it can be added later when it's implemented (the PR is still open that adds that message)

Reply:
Done!